gensim/models/nmf.py

```diff
@@ -141,15 +145,15 @@ def show_topics(self, num_topics=10, num_words=10, log=False, formatted=True):
 
         topics = self.get_topics()
 
-        print(topics)
+        # print(topics)
 
         for i in chosen_topics:
             topic = topics[i]
-            print(type(topic))
-            print(topic.shape)
+            # print(topic)
```

Comment:
Tip: comment temporary debugging / test lines with `# FIXME`. Easy grepping later, to make sure such stuff is not left behind and accidentally merged (or worse, left in active code paths).

Reply:
:ok_hand: